### PR TITLE
CMCL-1358: very rare axis drift

### PR DESCRIPTION
**Pull request by glabute**
### Purpose of this PR

CMCL-1358: Very difficult to reproduce, but real.  See detailed description in Jira issue.

### Testing status

- [ ] Added an automated test
- [ ] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [x] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Technical risk

low

### Comments to reviewers

This is difficult to test because the problem is extremely hard to reproduce in the first place.  It's an edge-case condition.  I've given the fix to the original poster on the forum, who is more easily able to repro.  They will confirm the fix.  I think the testing here should just be a manual reading of the code, and a checking that nothing was broken by the change.

### CODE REVIEW

File: com.unity.cinemachine/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased] 
 - Bugfix: AxisState was not respecting timescale == 0
+- Bugfix: Very occasional axis drift in SimpleFollow when viewing angle is +-90 degrees.
```
Comment (gaborkb) [inline com.unity.cinemachine/CHANGELOG.md:9]: The bugfix message is not correct based on the jira ticket description. You said that small movements are the culprit not the viewing angle.

**Review by gaborkb — APPROVED** [review]
Changelog text is outdated I think.

File: com.unity.cinemachine/CHANGELOG.md (hunk 1 above)
Comment (glabute) [reply com.unity.cinemachine/CHANGELOG.md:9]: It is correct.  It has to do with the viewing angle crossing the 90 degree boundary.

Comment (gaborkb) [reply com.unity.cinemachine/CHANGELOG.md:9]: From jira ticket:
"I was able to trigger this to happen with small mouse movements from any angle not just -90/90. So it seems like the angle is irrelevant other than the fact that if the bug is triggered the drift happens when viewing from the side and it won’t drift on its own when its near 0/180. Seems like just doing very small mouse movement rotations was enough to trigger it."

But I am ok with the changelog message, if you say it is correct. :)

Comment (glabute) [reply com.unity.cinemachine/CHANGELOG.md:9]: yeah, he meant "player angle in worldspace" which he at first thought was relevant, but it isn't
